### DISCUSSION TITLE
Can I use any directory to store Longhorn volumes or only full disks?

### DISCUSSION THREAD

**Discussion by trallnag** [longhorn/longhorn#2738] [Q&A] [answered]
According to <https://longhorn.io/docs/0.8.0/users-guide/multidisk/> I can have multiple disks. For example "fast-nvme" and "not-so-fast-ssd". Can I only add full disks or also any directory as a "disk"?

**Comment by cclhsu** [comment, marked as answer]
As described in https://longhorn.io/docs/1.1.1/volumes-and-nodes/multidisk/#add-a-disk, when adding a disk to Longhorn, you input the path mounted by the disk.

```
To add any additional disks, you need to:

Mount the disk on the host to a certain directory.
Add the path of the mounted disk into the disk list of the node.
```
